packages/terra-graphs/src/components/Bubble/BubbleMultipleDataset.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+import './BubbleMultipleDataset.module.scss';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+
+const propTypes = {
+  /**
+   * ID for graph created.
+   */
+  graphID: PropTypes.string.isRequired,
+  /**
+   * Graph configuration object
+   */
+  graphConfig: PropTypes.object.isRequired,
+  /**
+   * data to be displayed in graph
+   */
+  dataset: PropTypes.array,
+  /**
+   * timeout to display multiple data contents in specific time interval.
+   */
+  timeout: PropTypes.array,
+};
+
+const BubbleMultipleDataset = ({
+  graphConfig, dataset, graphID, timeout,
+}) => {
+  React.useEffect(() => {
+    const graph = Carbon.api.graph(graphConfig);
+    if (!(utils.isUndefined(dataset))) {
+      if (!(utils.isUndefined(timeout))) {
+        let count = 0;
+        dataset.forEach((data) => {
+          setTimeout(
+            () => (graph.graphContainer
+              ? graph.loadContent(Carbon.api.bubbleMultipleDataset(data))
+              : ''),
+            timeout[count],
+          );
+          count += 1;
```

Comment:
rather than defining and manually updating a count variable, we could just use the index of the loop
```suggestion
        dataset.forEach((data, index) => {
          setTimeout(
            () => (graph.graphContainer
              ? graph.loadContent(Carbon.api.bubbleMultipleDataset(data))
              : ''),
            timeout[index],
          );
```